sw/device/silicon_creator/lib/otbn_boot_services.h

```diff
@@ -40,19 +40,20 @@ rom_error_t otbn_boot_app_load(void);
  * P256 base point.
  *
  * The `additional_seed` is expected to be the output from a specially seeded
- * DRBG. It must be fully independent from the key manager seed.
+ * DRBG, and is provisioned into flash at manufacturing time. It must be fully
+ * independent from the key manager seed.
  *
  * Expects the OTBN boot-services program to already be loaded; see
  * `otbn_boot_app_load`.
  *
- * @param additional_seed Seed material from DRBG.
+ * @param additional_seed Which DRBG seed material to load from flash.
```

Comment:
nit: calling it a "DRBG seed" makes it sound like this value is the input for the DRBG, while if I understand correctly it's actually the output.

Reply:
Ah yes, sorry for the confusion, it is indeed the DRBG output (that was stored in flash at provisioning time). I re-worded this.